utils/parametric/_library_client.py

```diff
@@ -319,11 +320,14 @@ def trace_start_span(
         pb_links = []
         for link in links:
             pb_link = pb.SpanLink()
-            if (link.get("parent_id") or 0) == 0:
+            if link.get("parent_id") > 0:
                 pb_link.parent_id = link["parent_id"]
             else:
-                pb_link.http_headers = distributed_message
-                distributed_message = pb.DistributedHTTPHeaders()
+                link_headers = pb.DistributedHTTPHeaders()
```

Comment:
Currently only 1 span link can be added on span creation. This change to the Link object will allow us to initialize any number of span links via distributedhttpheaders. This also simplifies the span parenting logic in client apps.